x-pack/plugins/fleet/server/routes/epm/index.ts

```diff
@@ -57,6 +59,15 @@ export const registerRoutes = (router: IRouter) => {
     getLimitedListHandler
   );
 
+  router.get(
+    {
+      path: EPM_API_ROUTES.STATS_PATTERN,
+      validate: GetSummaryRequestSchema,
+      options: { tags: [`access:${PLUGIN_ID}`] },
+    },
+    getSummaryHandler
```

Comment:
again with the mixing of `STATS` and `Summary`

Reply:
My fault. I forgot to rename the methods. Working on it :)